vm/devices/net/netvsp/src/lib.rs

```diff
@@ -3382,6 +3406,21 @@ impl Adapter {
         Ok(())
     }
 
+    fn oid_set_packet_filter(
+        &self,
+        reader: impl MemoryRead + Clone,
+        primary: &mut PrimaryChannelState,
+    ) -> Result<bool, OidError> {
+        let filter: rndisprot::RndisPacketFilterOidValue = reader.clone().read_plain()?;
+        if filter != 0 {
```

Comment:
We need to interpret the filter value as specified by NDIS, at least to some extent.

To fully support the filter, we would need to distinguish between the different types of packets--unicast, broadcast, multicast, directed to the configured MAC vs. not. I don't think we should bother and we should just support "all on" or "all off".

So I think you could do something like `primary.stop_rx = filter == 0`.

Reply:
(But see elsewhere where I suggest saving the actual filter value.)